src/index.scss

```diff
@@ -5,7 +5,7 @@
 @import 'utilities/base';
 
 // Local
```

Comment:
I know this comment is not new, but It's kinda more a headline than a comment.

Reply:
Thank you for your notice. This issue has been taken care of.